pkg/network/ovn_kubernetes.go

```diff
@@ -46,6 +46,11 @@ const OVN_SHARED_GW_MODE = "shared"
 const OVN_LOG_PATTERN_CONSOLE = "%D{%Y-%m-%dT%H:%M:%S.###Z}|%05N|%c%T|%p|%m"
 const OVN_DPU_MODE_FULL = "full"
 const OVN_DPU_MODE_DPU_HOST = "dpu-host"
+const OVN_DPU_MODE_DPU = "dpu"
+const OVN_NODE_LABEL_DPU = "network.operator.openshift.io/dpu"
+const OVN_NODE_SELECTOR_DPU = "network.operator.openshift.io/dpu: ''"
+const OVN_NODE_LABEL_DPU_HOST = "network.operator.openshift.io/dpu-host"
+const OVN_NODE_SELECTOR_DPU_HOST = "network.operator.openshift.io/dpu-host: ''"
```

Comment:
you don't use the `OVN_NODE_LABEL_xxx` consts, and you don't use `OVN_NODE_SELECTOR_xxx` until the next commit so it probably should be added there, not in the error-cni commit.

Reply:
Moved `OVN_NODE_SELECTOR_DPU` to next commit and removed the rest. My plan was to replace all labels in the yaml files with these constants, but after doing so, thought it was too much abstraction and backed it out. Just using the one constant now.
Done.

Reply:
Moved `OVN_NODE_SELECTOR_DPU` to the next commit and removed the rest of the constants.
Original plan was to replace all the DPU base labels with these constants, but once implemented I decided it was too much abstract and back it out.